README.md

```diff
@@ -47,12 +56,47 @@ byName(name, name2 etc)	        gets a repo matching the name
 ```
 # List of Actions
 ```
-print-names:	                prints the name of the repo
-addTeam(organization,team):	    Adds a team from an organization to a repository
-clone(directory):               clones the repo to a directory/name-of-repo
-delete:                         deletes the repo
-listOpenPullRequests            lists open pull requests
-listOpenIssues                  lists open issues
+print-names:	                	prints the name of the repo
+addTeam(organization,team):	    	Adds a team from an organization to a repository
+clone(directory):               	clones the repo to a directory/name-of-repo
+delete:                         	deletes the repo
+listOpenPullRequests            	lists open pull requests
+listOpenIssues                  	lists open issues (NOT IMPLEMENTED)
+list-bugs-not-assigned				lists bugs not assigned
+list-issues-recently-closed			lists issues recently closed (1 day)
+list-issues-recently-updated		lists issues recently closed (1 day)
+list-issues-by-milestone(milestone)	lists issues matching argument milestone
+```
+
+# List of Issue Filters
+Note: These are not accesible from the command line, rather actions or other apps can build with them.
+```
+ALL 								matches all issues
+OPEN  								matches open issues
+CLOSED 								matches closed issues
+CLOSED_WITHIN_DAYS(numberDays)		matches issues closed within argument days	
+UPDATED_WITHIN_DAYS(numberDays)		matches issues updated within argument days
+HAS_LABEL(label)					matches issues containing argument label
+HAS_ASSIGNEE(login)					login of user, null argument returns issues with no assignee
+BY_MILESTONE(milestone)				filters by milestone
+BY_NAME								gets issue matching the name
 ```
+
+# Apps
+### DailyEmail.java
+Compiles and outputs both plain text (command line) and html (out_email.html) of users and key issues.
```

Comment:
Can you add instructions on how to run it from the command line, (i.e. after it has been compiled)